src/stacker/ConcreteStacker.ts

```diff
@@ -140,9 +142,43 @@ export class ConcreteStacker implements Stacker {
     commit: Commit,
   ): Promise<void> {
     const stack = [];
-    const nextCommits = [commit];
+    //TODO: Implement a complete version of the stack that start from the merge-base commit and also takes into consideration landed PRs
+    const mergeBaseCommitHashes = await this.sourceControl.getMergeCommitByCommitHash(
+      commit,
+    ); //Question: what if the commit is on the same branch as the mergeBaseCommit?
+    //Should one not be looking for a merge base commit then? Description update stopped working, reason? Did I fuck up something?
+    let parentCommitHashes = [...commit.parentCommits];
+    let currentCommitHash = commit.hash;
+    let baseCommit;
+    while (parentCommitHashes.length) {
+      const parentCommitHash = parentCommitHashes.pop();
+      const baseCommitHash = mergeBaseCommitHashes.find(
+        (commitHash) => commitHash === parentCommitHash,
+      );
+      if (baseCommitHash) {
+        baseCommit = await this.sourceControl.getCommitByHash(
+          currentCommitHash,
+        );
+        console.log("CURRENT COMMIT:", baseCommit);
```

Comment:
Is the console log here required?

Reply:
Thanks for pointing these out! Removing them.